src/ui-lib/sass/05-components/01-atoms/10-navigation/_00-block-link.scss

```diff
@@ -1,7 +1,6 @@
 .grav-c-block-link {
   display: inline-block;
-  padding: $grav-sp-inset-squished-xs;
-  padding-right: $grav-sp-m;
+  padding: $grav-sp-xs;
```

Comment:
Hmm... looking at this with a fresh pair of eyes, I've noticed a couple of things:

* This PR's code produces a larger visible gap between adjacent nav link texts than on the current Buildit site. This is because in this PR the gap is 2 * `$grav-ps-xs` padding + `$grav-sp-m` margin, whereas on the current Buildit site there is no margin between successive block links, so it's only the two lots of padding that create the space between words.
* The padding on block links is really a hangover from when they had the underline effect, but we've since removed that, so it's perhaps redundant now. A consequence of that is that the right-most link text is not flush with the boundary of the page's content area. Likewise, if the nav links wrap onto a new line, the left-most ones will also be inset by the padding.

I therefore propose the following:

* Give block links _only_ top and bottom padding (`$grav-sp-xs`). Arguably, this might be redundant too, but I feel it's worth keeping as it:
    * Makes the clickable/tappable link area a bit bigger and thus easier for users to hit (Fitts' Law)
    * Helps keep the overall layout of the nav links and header consistent with what's currently live on the Buildit website

This would achieve the following:

* The margin between adjacent block links in a nav menu that this PR introduces is equivalent to the two paddings in the current site, so visually the nav menu should look the same.
* Any links that are at the left or right edges of the page will have their link text flush with the edge of the content area, which I think will look a bit neater.

Reply:
I only just noticed this comment but I think I've addressed the issues as I also noticed this today. The only main difference is I kept the left/right padding and made them flush with negative margin on the parent. Seems to work well?